src/main/kotlin/uk/gov/justice/digital/hmpps/personrecord/api/model/canonical/CanonicalAddress.kt

```diff
@@ -1,20 +1,34 @@
 package uk.gov.justice.digital.hmpps.personrecord.api.model.canonical
 
+import io.swagger.v3.oas.annotations.media.Schema
 import uk.gov.justice.digital.hmpps.personrecord.jpa.entity.AddressEntity
 
 data class CanonicalAddress(
+  @Schema(description = "Person no fixed abode", example = "True")
   val noFixedAbode: String? = "",
+  @Schema(description = "Person address start date", example = "02/02/2020")
   val startDate: String? = "",
+  @Schema(description = "Person address end date", example = "04/04/2023")
   val endDate: String? = "",
+  @Schema(description = "Person address postcode", example = "B10 1EJ")
   val postcode: String? = "",
+  @Schema(description = "Person address sub building name", example = "Sub building 2")
   val subBuildingName: String? = "",
+  @Schema(description = "Person address building Name", example = "Main Building")
   val buildingName: String? = "",
+  @Schema(description = "Person address building number", example = "5")
   val buildingNumber: String? = "",
+  @Schema(description = "Person address thoroughfareName", example = "One Street")
```

Comment:
can we make this section a bit clearer?
Maybe give the address for MoJ Petty France or something?